extensions/scheduler/common/src/main/java/io/quarkus/scheduler/common/runtime/util/SchedulerUtils.java

```diff
@@ -174,7 +174,9 @@ private static long parseDurationAsMillis(Scheduled scheduled, String value, Str
     }
 
     private static Duration parseDuration(Scheduled scheduled, String value, String memberName) {
-        if (Character.isDigit(value.charAt(0))) {
+        if (Character.isDigit(value.charAt(0)) && Character.toLowerCase(value.charAt(value.length() - 1)) == 'd') {
```

Comment:
How about testing the first char only once, e.g. something like:
```java
if (Character.isDigit(value.charAt(0))) {
   if (value.toLowerCase().endsWith("d")) {
      value = "P" + value;
   } else {
      value = "PT" + value;
   }
}
```

Reply:
Thanks, will do. While there, do you think it would make sense to support milliseconds? something like "500ms". I don't think the worth is there as much, but for testing it can be nice to make the scheduled run faster sometimes so test can pass faster and who knows maybe it can also make sense in production. 

It would also be closer to what is done in the DurationConverter [here](https://github.com/quarkusio/quarkus/pull/32929). When reading the documentation, one could assume that the support for durations is the same everywhere in quarkus. WDYT?

Reply:
> Thanks, will do. While there, do you think it would make sense to support milliseconds? something like "500ms". I don't think the worth is there as much, but for testing it can be nice to make the scheduled run faster sometimes so test can pass faster and who knows maybe it can also make sense in production.

I don't think so. As [documented](https://quarkus.io/guides/scheduler-reference#intervals), _"A value less than one second may not be supported by the underlying scheduler implementation. In that case a warning message is logged during build and application start."_...